packages/sit-onyx/src/styles/grid.scss

```diff
@@ -0,0 +1,57 @@
+//
+// Default Grid Variants
+//
+
+// prettier-ignore
+$gridVariants: (
+//  NAME     BREAKPOINT  COLUMNS  GUTTER  MARGIN
+    "2xs"    0px         4        16px    16px,
+    "xs"     576px       8        16px    16px,
+    "sm"     768px       8        24px    32px,
+    "md"     992px       12       24px    64px,
+    "lg"     1440px      16       32px    64px,
+    "xl"     1920px      16       32px    64px,
```

Comment:
Shouldn't we use rem for gutter and margin?

Reply:
done